sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultServiceSpec.java

```diff
@@ -325,6 +323,7 @@ public boolean equals(ServiceSpec first, ServiceSpec second) {
                 DefaultVolumeSpec.class,
                 ExactMatcher.class,
                 HostnameRule.class,
+                InvalidPlacementRule.class,
```

Comment:
Given that we shouldn't be serializing configs with InvalidPlacementRule in them, is it possible to omit it from the classes available for deserialization? However if that causes other problems then leave it in.

Reply:
When not including this, the `ServiceTest` that I added to check this behaviour at a service level did not work - this may just be a side effect of the test or something that I am doing incorrectly.